client/src/pages/BoxExist.js

```diff
@@ -70,33 +58,41 @@ export default function BoxExist() {
 
   return (
     <>
-      <HeaderBackButton headline={box.title}></HeaderBackButton>
-      <ListContainer>
-        {box.items?.map((item) => (
-          <List
-            key={item}
-            item={item}
-            icon={IconMinus}
-            alt={"Icon Delete"}
-            onClick={() => handleDeleteItem(item)}
-          />
-        ))}
-      </ListContainer>
-      <ButtonContainer>
-        <FormInput
-          onSubmit={handleSubmitItem}
-          title="Hinzufügen"
-          icon={IconAdd}
-          alt="Icon add"
-          value={newItem}
-          onChange={(event) => setNewItem(event.target.value)}
-        />
-        <Button active onClick={() => setModalOpen(true)}>
-          <p>Schließen</p>
-          <img src={IconRecycle} alt="Icon recycle" />
-        </Button>
-      </ButtonContainer>
-      {modalOpen && <Modal boxtitle={box.title} closeModal={closeModal} />}
+      {status === "loading" && <div>Loading...</div>}
+      {status === "error" && <div>Es ist ein Fehler aufgetreten</div>}
```

Comment:
I would change the loading messages to German as well (or use a loading indicator icon)